mloop/learners.py

```diff
@@ -1999,20 +1990,20 @@ def update_bias_function(self):
         self.cost_bias = self.bias_func_cost_factor[self.params_count%self.bias_func_cycle]
         self.uncer_bias = self.bias_func_uncer_factor[self.params_count%self.bias_func_cycle]
 
-    def predict_biased_cost_unscaled(self,params):
+    def predict_biased_cost_unscaled(self,scaled_params):
```

Comment:
I just realized there may be a confusion about what constitutes "scaled" vs "unscaled". I typically use the convention that parameters are "scaled" whenever they are expressed in arbitrary units to have a range ~1 and "unscaled" when they are in actual units and have an arbitrary range. Here the parameters are expected to have been scaled, which is why we've called the argument `scaled_params`, so we're on the same page there.

The cost returned by `self.gaussian_process.predict()` is in arbitrary units with range ~1. Therefore I'd say the cost is scaled. To get the unscaled value we'd have to call the scaler's `inverse_transform()` method, which isn't done here. Therefore this method returns a scaled biased cost because it hasn't been un-scaled yet. That's why I think this method should be called `predict_scaled_biased_cost()`. This of course applies to the other renamed methods along these lines as well.

Reply:
I think I've addressed everything now. Thanks for the close check on the comments I've left, I'll be more careful with those in the future!